CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Changed
+
+- Pass RESTMapper to helmclient to reduce the number of REST API calls.
```

Comment:
```suggestion
- Pass RESTMapper to helmclient to reduce the number of REST API calls.
- Updated Helm to v3.5.3.
```

@tomahawk28 Can we add this?

So we don't need https://github.com/giantswarm/chart-operator/pull/727

Reply:
Sure, looks good 👍